lib/errors.js

```diff
@@ -0,0 +1,16 @@
+export class FilecoinRpcError extends Error {
+  /**
+   * @param {string} message
+   * @param {string} method
+   * @param {string} name
+   */
+  constructor(name, message, method) {
+    super(message)
+    this.name = name
+    this.method = method
+  }
```

Comment:
In JavaScript, `error.name` is the error class name, e.g. `FilecoinRpcError`.

A machine-readable description of the error is typically in `error.code`.

```suggestion
  /**
   * @param {string} code
   * @param {string} message
   * @param {string} method
   */
  constructor(code, message, method) {
    super(message)
    this.name = 'FilecoinRpcError'
    this.code = code
    this.method = method
  }
```